src/components/forms/ValidationErrors.js

```diff
@@ -2,17 +2,22 @@ import {FormFieldDescription} from '@utrecht/component-library-react';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-const ValidationErrors = ({error = ''}) => {
+const ValidationErrors = ({error = '', id = ''}) => {
```

Comment:
I think you want to leave this `undefined` instead of defaulting to an empty string so that if no `id` prop is passed, it doesn't get set. Otherwise this could possibly result in multiple divs with `<div id="">...</div>` and then IDs are not unique in the page.

The HTML 5 spec also says:

> When specified on [HTML elements](https://html.spec.whatwg.org/multipage/infrastructure.html#html-elements), the [id](https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute) attribute value must be unique amongst all the [IDs](https://dom.spec.whatwg.org/#concept-id) in the element's [tree](https://dom.spec.whatwg.org/#concept-tree) and must contain at least one character. The value must not contain any [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).

So unless React does something clever here, I'd stick to an explicit `id = undefined`

Reply:
Ah yeah fair point.. Yeah you're right, that shouldn't be an empty string 👍